core/src/main/scala/neotypes/implicits/mappers/ResultMappers.scala

```diff
@@ -100,17 +100,17 @@ trait ResultMappers extends ValueMappers {
                                              reprDecoder: Lazy[ResultMapper[R]],
                                              ct: ClassTag[A]): ResultMapper[A] =
     new ResultMapper[A] {
-      override def to(value: List[(String, Value)], typeHint: Option[TypeHint]): Either[Throwable, A] =
+      override def to(value: List[(String, Value)], typeHint: Option[TypeHint], errors: Seq[Throwable]): Either[Throwable, A] =
```

Comment:
I like to stay away from **Seq** whenever possible. I believe **List** would be the best here.

You can take a look to [this](https://gist.github.com/djspiewak/2ae2570c8856037a7738#generic-interfaces) to understand why I do not like to use generic sequences.

Reply:
Thanks for the link!  That's a great explanation